m4_sessions/1811_inning/big_tech.md

```diff
@@ -1,16 +1,17 @@
 ## Group Members:
-
+Ahmad Kayyali, Laura Shamus, Mike McKee, Matt Peters
 
 
 ## Name of Session: 
+Is big tech becoming too powerful?
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Big tech - good or bad?
```

Comment:
Give me a bit more of a description...Is Big Tech too powerful or will it save us all?  Come to this session and discuss!